### ISSUE TITLE
Angles regression is different in the paper and the code!

### ISSUE THREAD

**Issue by tfygg** [liulei01/DRBox#16]
In the paper, angles regression is used by equation (6c), I find the tan() is used! While sin() is used in the code!

Why? And which way is the best?

**Comment by keck91** [comment]
Where did you find this in the code?

**Comment by tjmannn** [comment]
yeah, I found this too, still confused, tan() seems to have same bug when angle deviation equals to 90

**Comment by tjmannn** [comment]
yeah, I found this too, still confused, tan() seems to have some bugs when angle deviation equals to 90
